suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -263,7 +263,10 @@ export const acquireDevice = createThunk(
  */
 export const authorizeDevice = createThunk(
     `${MODULE_PREFIX}/authorizeDevice`,
-    async (_, { dispatch, getState, extra }): Promise<boolean> => {
+    async (
+        { isUseEmptyPassphraseForced }: { isUseEmptyPassphraseForced: boolean },
```

Comment:
maybe you can make this optional and set default to false so diff will be minimal

Reply:
I wanted to do that, but if I make the `isUseEmptyPassphraseForced` optional, you need to pass empty object anyway.

I wanted to work it around by not passing the argument wrapped in the object, but rather as a simple boolean value. It would work that way, but then the linter shouts, that the optional parameter cannot be defined before the required one (`{dispatch, getState,...}`).

```
A required parameter cannot follow an optional parameter.ts(1016)
```

Reply:
Something like this shoud work:

```tsx
export const authorizeDevice = createThunk(
    `${MODULE_PREFIX}/authorizeDevice`,
    async (
        {
            isUseEmptyPassphraseForced = false,
        }: { isUseEmptyPassphraseForced?: boolean } | undefined = {},
        { dispatch, getState, extra },
    ): Promise<boolean> => {
```

I know it's little bit ugly but still better that doing diff in whole codebase IMHO

Reply:
Nice, that's exactly something I was looking for 👌